### PR TITLE
Windows MSIX instructions

### PR DESCRIPTION
**Pull request by domesticmouse**
Changes proposed in this pull request:
* Add notes about Microsoft's MSIX installer  
* Minor cleanups

fixes: https://github.com/flutter/flutter/issues/53167

### CODE REVIEW

File: src/desktop.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ toc: true
 Desktop support allows you to compile Flutter source code
 to a native Windows, macOS, or Linux desktop app. Flutter's desktop
 support also extends to plugins&mdash;you can install 
-existing plugins that support the macOS or Linux platforms,
+existing plugins that support the Windows, macOS or Linux platforms,
```
Comment (sfshaza2) [inline src/desktop.md:10]:
```suggestion
existing plugins that support the Windows, macOS, or Linux platforms,
```

File: src/desktop.md [hunk 2]
```diff
@@ -277,6 +278,27 @@ build output on other machines for testing purposes.
...
+For building Windows executables, you can either use tooling to construct an
+MSIX installer or you can build your own installer that collects together 
+the components.
+
+##### MSIX Packaging
+
+[MSIX][] is Microsoft's Windows app package format that provides a modern 
+packaging experience to all Windows apps. This format can either be used 
+to ship applications to Microsoft's Windows Apps store, or distribute 
+applications installers directly.
```
Comment (sfshaza2) [inline src/desktop.md:290]:
```suggestion
application installers directly.
```

File: src/desktop.md [hunk 3]
```diff
@@ -277,6 +278,27 @@ build output on other machines for testing purposes.
...
+
+The easiest way to create an MSIX distribution for a Flutter project is to use
+the [`msix` pub package][msix package]. For an example of using the `msix` package
+from a Flutter desktop app, see the [Desktop Photo Search sample][].
+
+[MSIX]: https://docs.microsoft.com/en-us/windows/msix/overview
+[msix package]: https://pub.dev/packages/msix
+[Desktop Photo Search sample]:https://github.com/flutter/samples/tree/master/experimental/desktop_photo_search
```
Comment (sfshaza2) [inline src/desktop.md:298]:
```suggestion
[msix package]: {{site.pub}}/packages/msix
[Desktop Photo Search sample]: {{github.com}}/flutter/samples/tree/master/experimental/desktop_photo_search
```

**Review by sfshaza2 — APPROVED** [review]
A few minor nits, but otherwise LGTM.

File: src/desktop.md (hunk 1 above)
Comment (sfshaza2) [inline src/desktop.md:10]:
```suggestion
existing plugins that support the Windows, macOS, or Linux platforms,
```

File: src/desktop.md [hunk 4]
```diff
@@ -277,6 +278,27 @@ build output on other machines for testing purposes.
 
 #### Windows
 
+For building Windows executables, you can either use tooling to construct an
+MSIX installer or you can build your own installer that collects together 
+the components.
```
Comment (sfshaza2) [inline src/desktop.md:283]:
```suggestion
MSIX installe, or you can build your own installer that collects 
the components together.
```

File: src/desktop.md (hunk 2 above)
Comment (sfshaza2) [inline src/desktop.md:290]:
```suggestion
application installers directly.
```

File: src/desktop.md (hunk 3 above)
Comment (sfshaza2) [inline src/desktop.md:298]:
```suggestion
[msix package]: {{site.pub}}/packages/msix
[Desktop Photo Search sample]: {{site.github}}/flutter/samples/tree/master/experimental/desktop_photo_search
```

**Review by sfshaza2 — APPROVED** [review]
A few minor nits, but otherwise LGTM.

**Comment by domesticmouse** [comment]
Thanks Shams!

Stuart and Chris, is this inline with your expectations for what we want to say about building windows installers?

**Review by cbracken — APPROVED** [review]
LGTM stamp from a Japanese personal seal

File: src/desktop.md [hunk 5]
```diff
@@ -277,6 +278,27 @@ build output on other machines for testing purposes.
...
+
+The easiest way to create an MSIX distribution for a Flutter project is to use
+the [`msix` pub package][msix package]. For an example of using the `msix` package
+from a Flutter desktop app, see the [Desktop Photo Search sample][].
+
+[MSIX]: https://docs.microsoft.com/en-us/windows/msix/overview
+[msix package]: {{site.pub}}/packages/msix
+[Desktop Photo Search sample]: {{site.github}}/flutter/samples/tree/master/experimental/desktop_photo_search
+
+##### Building your own installer for Windows
```
Comment (csells) [inline src/desktop.md:300]: These instructions aren't for building an installer -- they're for creating a zip file (which is a good thing to do but the title should reflect that).

Comment (domesticmouse) [reply src/desktop.md]: Done

**Pull request merged**